net/FlatBuffers/FlatBufferBuilder.cs

```diff
@@ -288,6 +313,34 @@ public void Add<T>(T[] x)
             Put(x);
         }
 
+#if ENABLE_SPAN_T
+        /// <summary>
+        /// Add a span of type T to the buffer (aligns the data and grows if necessary).
+        /// </summary>
+        /// <typeparam name="T">The type of the input data</typeparam>
+        /// <param name="x">The span to copy data from</param>
+        public void Add<T>(Span<T> x)
+            where T : struct
+        {
+            if (x.Length == 0)
```

Comment:
Is this worth special casing? If it is not a common occurrences, just letting Prep/Put do nothing may on average be faster and clearer.

Reply:
I removed the check.